commit ae0a338a10ef52f81ac2d85f8b977adec18be15e (refs/changes/93/5093/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-11-21T13:19:14-08:00 (11 months ago)
    
    TRITON-978 update Triton repos.json manifest of repos
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/repos.json b/repos.json
index 9f1e391..dd12cae 100644
--- a/repos.json
+++ b/repos.json
@@ -80,6 +80,9 @@
         {
             "name": "fsnotify"
         },
+        {
+            "name": "grafana"
+        },
         {
             "name": "java-triton"
         },
@@ -896,6 +899,9 @@
                 "vm": true
             }
         },
+        {
+            "name": "triton-mockcloud-common"
+        },
         {
             "name": "triton-origin-image"
         },
@@ -931,7 +937,6 @@
     ],
     "excludedRepositories": [
         "Cassandra-Operator",
-        "Deployment-Audit-Tool",
         "MALT",
         "MANTA-3788",
         "Manta-Client",
@@ -969,10 +974,12 @@
         "cosbench-manta",
         "cosbench2csv",
         "cr-update-reviewers",
+        "ctf2json",
         "daggr",
         "dcim_agent",
         "debian-image-tests",
         "debian-lx-brand-image-builder",
+        "deployment-audit-tool",
         "docker-machine-driver-triton",
         "dogpatch",
         "dragnet",
@@ -1040,6 +1047,7 @@
         "lx-brand-image-tests",
         "lx-tests",
         "machine",
+        "mako-regional-report",
         "manatee-state-machine",
         "manifest-creator",
         "manta",
@@ -1126,6 +1134,7 @@
         "node-codel",
         "node-consulite",
         "node-contract",
+        "node-ctype",
         "node-debug-school",
         "node-debugging-methodologies",
         "node-dns",
